elasticsearch/helpers/vectorstore/_async/strategies.py

```diff
@@ -285,44 +285,50 @@ def _hybrid(
         # RRF is used to even the score from the knn query and text query
         # RRF has two optional parameters: {'rank_constant':int, 'rank_window_size':int}
         # https://www.elastic.co/guide/en/elasticsearch/reference/current/rrf.html
-        rrf_options = {}
-        if isinstance(self.rrf, Dict):
-            if "rank_constant" in self.rrf:
-                rrf_options["rank_constant"] = self.rrf["rank_constant"]
-            if "window_size" in self.rrf:
-                # 'window_size' was renamed to 'rank_window_size', but we support
-                # the older name for backwards compatibility
-                rrf_options["rank_window_size"] = self.rrf["window_size"]
-            if "rank_window_size" in self.rrf:
-                rrf_options["rank_window_size"] = self.rrf["rank_window_size"]
-        query_body = {
-            "retriever": {
-                "rrf": {
-                    "retrievers": [
+        standard_query = {
+            "query": {
+                "bool": {
+                    "must": [
                         {
-                            "standard": {
-                                "query": {
-                                    "bool": {
-                                        "must": [
-                                            {
-                                                "match": {
-                                                    self.text_field: {
-                                                        "query": query,
-                                                    }
-                                                }
-                                            }
-                                        ],
-                                        "filter": filter,
-                                    }
-                                },
-                            },
-                        },
-                        {"knn": knn},
+                            "match": {
+                                self.text_field: {
+                                    "query": query,
+                                }
+                            }
+                        }
                     ],
-                    **rrf_options,
-                },
-            },
+                    "filter": filter,
+                }
+            }
         }
+
+        if self.rrf is False:
+            query_body = {
```

Comment:
With `rrf=False` we have to use a standard query, since the retriever query can only combine results from multiple searches with RRF.